docs/design-system/ui_kits/ovp/ovp-ui.css

```
@@ -0,0 +1,268 @@
+  --accent-2: #2563bb; --accent-2-soft: #dbe7f7;
+  --warn-border: #d48a2f; --warn-bg: #fff8ec; --warn-text: #a85e10;
+  --code-bg: #f4f4f5; --code-text: #1f1a17;
+  --graph-bg: #efece5; --graph-link: rgba(31,26,23,0.16);
+  --graph-link-hi: #9f4f24;
+  --graph-hull: rgba(159,79,36,0.10); --graph-hull-edge: rgba(159,79,36,0.55);
+  --graph-grid: rgba(31,26,23,0.04);
+  --c-1:#9f4f24; --c-2:#2563bb; --c-3:#2f7d52; --c-4:#b8862e;
+  --c-5:#6c4dab; --c-6:#b04545; --c-7:#2d6e87; --c-8:#645a52;
+  --shadow-shell: 0 12px 36px rgba(31,26,23,0.06);
+  --shadow-pop: 0 16px 48px rgba(31,26,23,0.12);
+  --graph-bg: radial-gradient(ellipse at 50% 40%, #fdfaf3 0%, #efece5 60%, #e2ddd2 100%);
```

Stylelint (17.10.0)

[error] Duplicate custom property "--graph-bg" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)